app/extensions/settings/components/settings/DbSettings.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace justcoded\yii2\settings\components\settings;
```

Comment:
please remove folder "settings".
it's useless, because we develop independent component "settings".